troposphere/static/js/actions/ImageActions.js

```diff
@@ -1,14 +1,25 @@
 define(function (require) {
 
-  var AppDispatcher = require('dispatchers/AppDispatcher'),
+  var moment = require('moment'),
+      AppDispatcher = require('dispatchers/AppDispatcher'),
       ImageConstants = require('constants/ImageConstants');
 
   return {
 
     updateImageAttributes: function (image, newAttributes) {
+      if(newAttributes.end_date != null) {
+        var end_date;
+        if (typeof newAttributesend_date == "object") {
```

Comment:
is this a typo? `newAttributesend_date`? 

I would expect this to be: 
```javascript
if (typeof newAttributes.end_date == "object") {
  ...
}
```

Also, this might be a bit of "belt && suspenders" - but I think we want ``===`` here to be safe ([typeof](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof) examples).